src/checkstyle/fixRightCurly.gradle

```diff
@@ -0,0 +1,65 @@
+task fixRightCurly << {
+	fileTree("${buildDir}/reports/checkstyle").include('*.xml').each { report ->
+		def xml = new XmlParser(false, false).parse(report)
+		xml.file.each { f ->
+			def errors = f.error
+			def thisErrors = []
+			errors.each { error ->
+				if (error.@source == 'com.puppycrawl.tools.checkstyle.checks.blocks.RightCurlyCheck') {
+					thisErrors.add(error)
```

Comment:
I think we'll end up removing all these when we're done so not worth too much polish here.